tests/ui/attributes/malformed-attrs.rs

```diff
@@ -213,7 +213,8 @@ extern crate wloop;
 //~^ ERROR can't find crate for `wloop` [E0463]
 
 #[macro_export = 18]
-//~^ ERROR malformed `macro_export` attribute input
+//~^ ERROR valid forms for the attribute are
+//~| WARN this was previously accepted by the compiler
```

Comment:
I think this isn't very nice. It took me a while to see that this really was a problem, tried some things to make sure. But this was already an error, not a warning. Only the `#[macro_export(meow)]` was a warning that will become an error. For this form, the compiler already gave an error *and* now we're getting an error, plus also a warning that it'll be an error in the future. I don't think that's right